packages/validation/backend/sirius-components-validation-graphql/.settings/org.springframework.ide.eclipse.boot.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+spring.boot.launch.profile.history=dev;
```

Comment:
Is this pref file necessary ? I can see that some projects have it (`sirius-components-diagrams`), and some don't (`sirius-components-core`).

Reply:
It's not really necessary, it only contains the profile used while launching the code of the project. I suspect that it's related to the launch of unit tests and the final application like Sirius Web. I would really like not to synchronize that so I'll remove it.